README.md

```diff
@@ -6,6 +6,7 @@
 [![Test Coverage](https://img.shields.io/codecov/c/github/fecgov/openFEC/master.svg)](https://codecov.io/github/fecgov/openFEC)
 [![Code Climate](https://img.shields.io/codeclimate/github/fecgov/openFEC.svg)](https://codeclimate.com/github/fecgov/openFEC)
 [![Known Vulnerabilities](https://snyk.io/test/github/fecgov/openFEC/badge.svg)](https://snyk.io/test/github/fecgov/openFEC)
+[![Known Vulnerabilities](https://snyk.io/test/github/fecgov/openfec/badge.svg?targetFile=data%2Fflyway%2Fbuild.gradle)](https://snyk.io/test/github/fecgov/openfec?targetFile=data%2Fflyway%2Fbuild.gradle)
```

Comment:
@patphongs Since these have the same labels, they look like duplicates. Maybe we should give them different labels, like "API Vulnerabilities" and "Flyway vulnerabilities"?

![image](https://user-images.githubusercontent.com/31420082/41791875-10439cda-7625-11e8-839c-cb4940ca7224.png)